plugins/osdn/subnets.go

```diff
@@ -123,21 +128,23 @@ func (oc *OvsController) SubnetStartNode(mtu uint) (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	subnets := result.([]api.Subnet)
+	subnets := result.([]osapi.HostSubnet)
 	for _, s := range subnets {
-		oc.flowController.AddOFRules(s.NodeIP, s.SubnetCIDR, oc.localIP)
+		if s.HostIP != oc.localIP {
+			oc.pluginHooks.AddHostSubnetRules(&s)
```

Comment:
If OVS is totally messed up on the machine, then we'd get errors from the OVS methods during initial setup, and those errors do get checked.

If we fail here because of a bug (syntax error in the rule, bug in pkg/ovs, etc) then that should make networking not work correctly, which should then get caught by the unit tests and then fixed, so the failure would never happen in the real world.

So basically, errors from these functions are sort of "can't happen". The functions used to return errors which all callers always ignored; it seemed to make more sense to remove the errors than to make the callers check them...